src/commands/setup.yml

```diff
@@ -0,0 +1,31 @@
+description: >
+      Create ECR repository for docker image out of the project and create SQS queues for deployments
+      $AWS_ACCOUNT_ID must be set
+
+parameters:
+      env:
+            description: target environment (i.e, qa, staging, production)
+            type: string
+      project:
+            description: your git repository project name
+            type: string
+      bucket:
```

Comment:
I is a bit not clear what this means... think of a developer trying to set up a new project.
Maybe this can be derived from the project name ? or env ?

Reply:
we can use CIRCLE_PROJECT_REPONAME, but other orbs have this prop too, so need to change all other orbs to use this variable to be consistent.